src/components/CommentList.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react";
+import { CommentCard } from "./CommentCard";
+import { getCommentsByArticle } from "../../api";
+import { useParams } from "react-router-dom";
+
+export function CommentList() {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    getCommentsByArticle(article_id).then((response) => {
+      setComments(response);
+      setIsLoading(false);
+    });
+  }, []);
+
+  if (isLoading) {
+    return <h1>Loading...</h1>;
+  }
+
+  return (
+    <ul>
+      {comments.map((comment) => {
+        return (
+          <li className="comment_card" key={comment.comment_id}>
+            <CommentCard comment={comment} />
+          </li>
+        );
+      })}
+    </ul>
+  );
+}
```

Comment:
This whole Comments component is not part of this ticket. Please ensure you are only focusing on what the ticket contains